other/pod_lifetime_annotation/policy.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: kyverno.io/v1
+kind: ClusterPolicy
+metadata:
+  name: pod-lifetime
+  annotations:
+    policies.kyverno.io/title: Enforce pod duration
+    policies.kyverno.io/category: Sample
+    policies.kyverno.io/minversion: 1.4,2
+    policies.kyverno.io/subject: Pod
+    policies.kyverno.io/description: >-
```

Comment:
See sister comment on your kyverno/website PR, but one other thing we're trying to standardize on with this `description` annotation is to state two things: 

1. Why is such a policy valuable?
2. What does this policy do?

You cover the second one but if you would please put one or two short sentences here for why this policy is useful.

Reply:
Added some extra description to cover why this is valuable

Reply:
"This validation is valuable with annotations are used..."

Should this be "when annotations are used"?

Reply:
Fixed wording